### DISCUSSION TITLE
Blink onboard LED on Pico W using PIO?

### DISCUSSION THREAD

**Discussion by mdaeron** [micropython/micropython#10881] [RP2040 / Pico] [answered]
Sorry if this is obvious, but I can't seem to find a way to blink the Pico W's onboard LED using the PIO.

`rp2.StateMachine(0, blink, freq=2000, set_base=machine.Pin('LED'))` yields the following error: `ValueError: expecting a regular GPIO Pin`.

Is there an obvious workaround that I'm missing?

**Comment by dpgeorge** [comment, marked as answer]
This is not possible to do on the Pico W, because on that board the built-in LED is driven by the WiFi chip (the CYW4343).  So there's quite a lot going on to turn that LED on/off, it's not a simple GPIO on the RP2040.

**Reply by mdaeron** [reply]
Thanks!  I assumed (wrongly) that the simplest way to tech myself to write a PIO program was to blink that LED. No worries.
